networkit/test/test_centrality.py

```diff
@@ -4,11 +4,92 @@
 import networkit as nk
 import numpy as np
 import scipy
-
+import random
 
 class Test_Centrality(unittest.TestCase):
```

Comment:
Sry for the inconvinience - can you please rename the classes to UpperCamelCase?